OpenSourcePolitics/decidim-app
# feat: Send SMS Reminder

#### :tophat: Description
Add rake task to generate a temporary csv file with users that have pending votes and send them a sms reminder

#### :pushpin: Related Issues
- [Notion card](https://www.notion.so/opensourcepolitics/Marseille-Relance-SMS-vote-80cfac3ed2b2490699f718fa3709d72c?pvs=4)

#### Testing
- Access to your platform
- Try to vote on the first budget (of the first process) 
- Don't finish your vote
- Access to your terminal
- Launch the command `BUDGET_ID=1 bundle exec rake decidim_app:budgets:send_sms_reminder`
- Make sure you read in your terminal the id of your user 
- Please note that you may have to change the BUDGET_ID until you find something

PS : You must search something like this in your terminal `"users ids: [{your_user_id}]"`

This should send a sms to users who have pending votes for a specific budget
